### ISSUE TITLE
Regarding Potential RS485 Support

### ISSUE THREAD

**Issue by Ananyaaynana** [LibreSolar/bms-firmware#62]
Are there any plans to incorporate RS485 functionality in future releases?
If such plans exist, is there a rough timeframe or priority level associated with their implementation?

The inclusion of RS485 capabilities would greatly enhance the project's versatility for applications requiring communication over longer distances or in multi-drop configurations.

**Comment by martinjaeger** [comment]
No, it's currently not planned. Just added the hardware capabilities so that it can be added when needed.

I can offer to implement it as part of a small contract. If that is something you would consider, please get in contact with me via E-Mail.
